src/test/java/com/github/cchacin/cucumber/steps/example/app/Controller.java

```diff
@@ -99,6 +114,20 @@ public Response successfulPOST(final String body) {
         return Response.created(uriInfo.getAbsolutePathBuilder().path("1").build()).build();
     }
 
+    @PUT
+    @Path("/successful/headers/put")
+    @Consumes("application/json")
+    public Response successfulHeadersPUT(final String body) {
+        return Response.noContent().build();
```

Comment:
Can you PUT the request headers in the response and check it in the feature file?

Just to make sure the we are sending the headers properly